### ISSUE TITLE
Implement ds-math

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/minime#10]
https://github.com/Giveth/ds-math/blob/master/src/SafeMath.sol

I like this library! lets use it (Thx Dani ;-) )

**Comment by gjgd** [comment]
Agreed, there are a some places in the code where overflow may happen.
Eg https://github.com/Giveth/minime/blob/master/contracts/MiniMeToken.sol#L437
